package.json

```diff
@@ -15,25 +16,25 @@
     "aggregate-error": "^3.0.0",
     "debug": "^4.0.0",
     "execa": "^5.0.0",
-    "lodash": "^4.17.4",
-    "parse-json": "^5.0.0"
+    "lodash-es": "^4.17.21",
+    "parse-json": "^5.0.0",
+    "prettier": "^3.3.3"
   },
   "devDependencies": {
     "ava": "5.1.0",
-    "nyc": "15.1.0",
-    "semantic-release": "21.1.2",
+    "c8": "^10.1.2",
+    "semantic-release": "^24.1.2",
```

Comment:
super minor since renovate will pin these after they are merged anyway, but more for awareness, our typical practice is to pin dev-deps to exact versions and leave prod-deps as ranges

Reply:
Sounds good... Well noted